### PR TITLE
little review

### PR DESCRIPTION
**Pull request by QuentinPerez**

### CODE REVIEW

**Comment by moul** [comment]
LGTM

File: pkg/commands/commit.go [hunk 1]
```diff
@@ -33,6 +33,6 @@ func RunCommit(ctx CommandContext, args CommitArgs) error {
 	if err != nil {
 		return fmt.Errorf("Cannot create snapshot: %v", err)
 	}
-	fmt.Println(snapshot)
+	fmt.FPrintln(ctx.Stdout, snapshot)
```
Comment (moul) [inline pkg/commands/commit.go]: `FPrintln` -> `Fprintln`

**Comment by QuentinPerez** [comment]
Done 😉

**Comment by moul** [comment]
LGTM

**Pull request merged**